lib/ex_doc/formatter/html/templates.ex

```diff
@@ -157,7 +172,19 @@ defmodule ExDoc.Formatter.HTML.Templates do
   end
 
   def module_summary(module_node) do
-    [Types: module_node.typespecs] ++
+    sections =
+      case module_node.rendered_doc do
+        nil ->
+          []
+
+        moduledoc ->
+          Enum.map(extract_headers(moduledoc), &Map.put(&1, :type, :heading))
+      end
+
+    [
+      Sections: sections
+    ] ++
```

Comment:
I don't know how tricky it would be to do this, but can we included the Sections before the Summary? Since they appear before the Summary on the page.

Reply:
Done!